README.md

```diff
@@ -356,7 +356,7 @@ database is encountered that is not prefixed then the primary metastore is used
       remote-meta-store-uris: thrift://primaryLocalMetastore:9083
     federated-meta-stores:
       - name: federated
-        prefix: waggle_prod_
+        database-prefix: waggle_prod_
```

Comment:
Oh dear, was this wrong in our documentation all along?

Reply:
oops

Reply:
😅